src/line.rs

```diff
@@ -1160,6 +1169,23 @@ impl<'input> FileEntry<'input> {
         self.directory_index
     }
 
+    /// Get this file's directory.
+    ///
+    /// If this file's directory index is 0, meaning that it was found in
+    /// the current directory of compilation, return `None`.
+    pub fn directory<Endian>(&self,
+                             header: &LineNumberProgramHeader<'input, Endian>)
```

Comment:
I'd like this API to take a UnitHeader or something and attempt to parse out the current directory, too, but I think this can be done in a follow up.